elm-street.cabal

```diff
@@ -75,9 +75,9 @@ library
 
   build-depends:       aeson >= 1.3
                      , directory ^>= 1.3
-                     , filepath ^>= 1.4
+                     , filepath >= 1.4 && < 1.6
                      , prettyprinter >= 1.2.1 && < 1.8
-                     , text >= 1.2 && <= 2.1
+                     , text >= 1.2 && <= 2.2
```

Comment:
is strict upper bound really that useful here?

Reply:
What do you propose?
Using some arbitrary other bound?
Dropping the upper bound entirely?

Reply:
I'm not really sure. Maybe we can stretch this to < 3.0 ?